src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityTokenHandler.cs

```diff
@@ -276,6 +297,24 @@ private ClaimsPrincipal ValidateToken(Saml2SecurityToken samlToken, string token
             return new ClaimsPrincipal(identity);
         }
 
+
+        private static async Task<TokenValidationParameters> PopulateValidationParametersWithCurrentConfigurationAsync(TokenValidationParameters validationParameters)
```

Comment:
This should be a shared method between Saml and Saml2.

Reply:
Fixed 😄